cardano-testnet/src/Testnet/Process/Run.hs

```diff
@@ -193,6 +195,16 @@ mkExecConfig tempBaseAbsPath sprocket networkId = do
     , H.execConfigCwd = Last $ Just tempBaseAbsPath
     }
 
+-- | Adds environment variables to an 'ExecConfig' that may already
+-- have some environment variables set. This is done by prepending the new
+-- environment variables to the existing ones.
+addEnvVarsToConfig :: H.ExecConfig -> [(String, String)] -> H.ExecConfig
```

Comment:
We should modify `mkExecConfig` to accept a list of environment variables.

Reply:
In this case, it is good to have it locally modified though, cause it is only relevant inside the `serveFilesWhile`

Reply:
Or we could expose lenses for that purpose?

Reply:
It sounds like a good idea, some kind of `Prism`. But I am not sure how that would behave with the `Last` and the `Maybe`. For example, if it is `Nothing` it wouldn't be just mapping over it, so I am going to leave it for another time for now...